packages/docusaurus-plugin-content-docs/src/__tests__/index.test.js

```diff
@@ -238,11 +238,9 @@ describe('loadDocs', () => {
       {
         path: '../docs',
         sidebarPath,
+        skipNextRelease: true,
```

Comment:
Good move. But before we commit on this we must be sure that most users will not need this to be environment dependent.